### PR TITLE
Fixes #20163 - added libselinux and missing deps

### PR DESCRIPTION
**Pull request by lzap**
Taken out from

https://github.com/theforeman/foreman-discovery-image/pull/94

we need to add some more dependencies which is unfortunate, ISO file will grow a bit again.

In the future we want to try Atomic as our base.

### CODE REVIEW

**Comment by lzap** [comment]
I will add the updated script as well.

**Comment by lzap** [comment]
Done @dLobatog or @timogoebel mind taking a look? I need this downstream, does not build against RHEL or CentOS 7.4 beta.

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ root/opt/extension/facts/*
 root/opt/extension/lib/ruby/*
 root/opt/extension/lib/*
 root/opt/extension/bin/*
+fdi-rhel7.ks
+00-repos-rhel7.ks
```
Comment (lzap) [inline .gitignore]: These files are not commited as they contain internal HTTP links into brew and won't be useful.

File: 20-packages.ks [hunk 2]
```diff
@@ -77,9 +77,14 @@ e2fsprogs
...
 
-#
-# Packages to Remove
-#
+######################
+# Packages to Remove #
+######################
+
+# Red Hat Enteprise Linux subscription tool
+-subscription-manager
```
Comment (lzap) [inline 20-packages.ks]: We used to have it downstream but now it needs more dependencies so let's get rid of it.

**Comment by lzap** [comment]
Done @dLobatog or @timogoebel mind taking a look? I need this downstream, does not build against RHEL or CentOS 7.4 beta.

**Comment by lzap** [comment]
FDI downstream build is currently blocked by this @ehelms

**Comment by dLobatog** [comment]
Thanks @lzap !

**Pull request merged**